source/CardGames.py

```diff
@@ -324,28 +335,81 @@ def display_board_general(self, cards_in_play):
 
 
     def fugitive_first_turn(self):
-      fugitive_deck = self.starting_cards
-      fugitive_deck.append(self.escape_card)
+      self.fugitive_deck = self.starting_cards
+      self.fugitive_deck.append(self.escape_card)
       for x in range(3):
-        fugitive_deck.append(self.LowRangeDeck.pop())
+        self.fugitive_deck.append(self.LowRangeDeck.pop())
 
       for x in range(2):
-        fugitive_deck.append(self.MidRangeDeck.pop())
+        self.fugitive_deck.append(self.MidRangeDeck.pop())
 
       string = ""
-      for i in fugitive_deck:
+      for i in self.fugitive_deck:
         string += str(i.value) + ", "
 
       string = string[:len(string)-2]
       print("Here is your starting hand: " + string) 
 
       #May have to check if burn is empty string in case fugitive does not want to burn anything
-      burn = input("Enter which cards to burn separated only by a comma (1,2,3...)").split(',')
-      hideouts = input("Select two viable cards you want to place as hideouts separated only by a comma (1,2,3...): ").split(',')
+      #burn = input("Enter which cards to burn separated only by a comma (1,2,3...)").split(',')
       #Would probably call function to check if hideouts are viable here. If hideouts aren't viable, reprompt fugitive
 
-      return burn, hideouts
-      
+      #return fugitive_deck, hideouts
+      print(self.fugitive_deck)
+
+    #check if selected hideout is valid for first turncd
+    def check_illegal_card(self):
```

Comment:
Let's try to make it work with Ethan's

Reply:
Yes, I agree. We need to meet and go through the logic of the code base.